app/models/shelter.rb

```diff
@@ -31,4 +32,13 @@ def alphabetical_pets
   def shelter_pets_filtered_by_age(age_filter)
     adoptable_pets.where("age >= ?", age_filter)
   end
+
+  # RAW SQL
+  def self.order_by_reverse_alphabetical
```

Comment:
These look good, just make sure there are model tests for them as well!! I believe model tests for these methods are missing